.github/workflows/build_and_test.yaml

```diff
@@ -258,7 +258,3 @@ jobs:
         uses: ./.github/workflows/collect-logs
         with:
           name: canary
-        
-      - name: consider debugging
-        uses: lhotari/action-upterm@v1
-        if: failure()
```

Comment:
I'm way more interested in enabling this via debug than removing the functionality entirely; but I do agree that it needs to take up less time. Worth mentioning, too, that this will need updating in a couple more places (I believe it's used in three different workflows at the moment)

Reply:
the latest change removes another of the consider-debugging steps, rather than checking for debug